functions

```diff
@@ -428,7 +440,10 @@ add_module() {
         ext4)
             add_module "crypto-crc32c?"
```

Comment:
Why bother recursing? Just inline this as `_add_module+=(...)`. Don't forget to normalize `nls_iso8859-1`.

Reply:
Recursing here so that the above logic skips potentially duplicate modules.